lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,56 @@
 import React from 'react';
+import PropTypes from 'prop-types'
+import styled from 'styled-components';
+
+
+const TabDiv = styled.div`
+display: flex;
+  justify-content: none;
+  align-items: center;
+  flex-direction: row;
+  color: #fff;
+  background-color: #333;
+  margin: 0 5px;
+  padding: 2px 10px;
+  font-size: 12px;
+  letter-spacing: 2px;
+  cursor: pointer;
+  font-weight: bold;`
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+     
+      let changeClass = '';
+
+      if (props.tab === props.selectedTab) {
+
+        changeClass = 'tab active-tab';
+
+        } else {
+
+        changeClass = 'tab';
+
+      }
+
   return (
-    <div
-      className={''}
-      onClick={() => {
+    <TabDiv
```

Comment:
a good way of implementing the if statement above is to actually use this pattern called conditional rendering for react. To do conditional rendering, you need to use a ternary operator like so: 

`className={props.tab === props.selectedTab ? "tab active-tab" : "tab"}`

the condition followed by the question mark. If condition is true, the first one will fire. : is the else